soundbay/data.py

```diff
@@ -163,9 +167,17 @@ def _get_audio(self, path_to_file, begin_time, end_time, label):
                 start_time = random.randint(begin_time - margin_len_begin, end_time - margin_len_end)
             else:
                 start_time = random.randint(begin_time, end_time - int(self.seq_length * self.data_sample_rate))
+            if start_time < begin_time:
```

Comment:
You actually turn off here the margin_len feature, not sure it's a good idea
what was the original bug that raised and issue?